exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,8 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+function greetingUpperCase() {
+    return 'hello ' + mentor3;
+}
+let greeting = greetingUpperCase();
+console.log(greeting.toUpperCase())
```

Comment:
GREAT. I learn from this one I did in different way(I use twice toUpperCase) I will change mine to yours